src/auth/auth.service.ts

```
@@ -182,5 +182,44 @@ export class AuthService {
     return await this.userRepository.findOne({ where: { email } });
   }
   
+  async getUsersWithFilters(page: number, limit: number, role?: UserRole, isSuspended?: boolean) {
```

ESLint

[error] Replace `page:·number,·limit:·number,·role?:·UserRole,·isSuspended?:·boolean` with `⏎····page:·number,⏎····limit:·number,⏎····role?:·UserRole,⏎····isSuspended?:·boolean,⏎··`

(prettier/prettier)